roles/confluent.test/molecule/mtls-debian/molecule.yml

```diff
@@ -122,6 +134,8 @@ provisioner:
   inventory:
     group_vars:
       all:
+        default_internal_replication_factor: 4
```

Comment:
Oh one more thing, can you add some tasks to the verify.yml of this scenario that validates the variable does what you expect? We have a helper task file called check property.

You can run the verify.yml playbook after you converge with: 
```
molecule verify -s mtls-debian
```

Reply:
added.

ps: as now jolokia_enabled=false by default, molecule tests that expect jolokia available will fail. will create a issue to track this.

Reply:
https://github.com/confluentinc/cp-ansible/issues/532